Source/Bugsnag.h

```diff
@@ -27,6 +27,7 @@
 
 #import "BugsnagConfiguration.h"
 #import "BugsnagMetaData.h"
+#import "BugsnagPlugin.h"
```

Comment:
Is this import required?

Reply:
Yes. The header is a part of the public module export, so it has to be included in the umbrella header (Bugsnag.h).